packages/components/src/Filter/FilterBar/context/types.ts

```diff
@@ -32,6 +32,7 @@ export type FilterBarState<ValuesMap extends FiltersValues> = {
   activeFilterIds: Set<keyof ValuesMap>
   values: Partial<ValuesMap>
   dependentFilterIds: Set<keyof ValuesMap>
+  focusId?: keyof ValuesMap | undefined
```

Comment:
is the "?" here enough to assert that focusId could be undefined?

Reply:
Yep nice catch